src/wiki/ventoy.md

```diff
@@ -0,0 +1,106 @@
+
+2. The next screen you see will be this, wait for the installer to complete preparation.
+4. Use the recommended settings, or select your preferred Language, Edition, and Architecture.
+   ![](./img/downloading-windows/gettingthings.png)
+::: tip Note
+If you're planning to install Windows on another PC and don't know what architecture it uses, then select "Both".
+:::
+   — You need to click on:**want to reinstall Windows on another computer** click **Create installation media**.
+
+    ![](./img/downloading-windows/whatdoyouwanttodo.png)
+5. Select **ISO file**, then press next.
+4. Select the correct Architecture, Edition, and Language of Windows, or use the recommended settings.
+
+   ::: tip Note
+   If you're planning on installing Windows on another PC and don't know what to select for the architecture, then choose both x86 and x64.
+   :::
+
+   ![](./img/downloading-windows/changelanguagearchitecture.png)
+Once the file has been downloaded, copy it to the Ventoy Drive.
+5. On the Media Creation Tool, select **ISO file**.
+
+You can read more about reinstalling Windows [here](installing-windows).
+
+6. Select a path to save to.
+## Downloading Antiviruses
+If you want to detect and remove any malware or rootkits that your computer has been infected with, you can try the following options.
+
+ - Microsoft Defender Offline: [32-bit](https://go.microsoft.com/fwlink/?LinkID=234123), [64-bit](https://go.microsoft.com/fwlink/?LinkID=234124)
+
+Download the ISO, and copy it to your Ventoy Drive.
+The instructions for installing or reinstalling Windows in your pc are here: [Installing Windows](installing-windows).
+:::
+
+## Download the AV
+If you want to detect rootkits and malware that got infected in your computer, and you want to clean them, use you can use one you've here listed:
+ - Kaspersky: https://www.kaspersky.com/downloads/free-rescue-disk (Not usable in US).
+ - Microsoft Defender Offline: https://support.microsoft.com/en-us/windows/help-protect-my-pc-with-microsoft-defender-offline-9306d528-64bf-4668-5b80-ff533f183d6c (head into Windows 8.1 and 7 to know how to download your .iso).
+
+With the ISO, send it to your Ventoy USB.
+
+After that, copy the ISO file to your Ventoy Drive.
+The usage is very simple, just you click to update the antivirus databases and run the scan following the instructions on your screen.
+:::
+Instructions on using Clonezilla are available [here](https://clonezilla.org//clonezilla-live-doc.php).
+## Clonezilla
+Clonezilla is great software for managing (backup and restore, clone)
+Download the ISO here, selecting your architecture (amd64:  x86_64/
+i686=32-bit) and selecting ISO as file type: https://clonezilla.org//downloads/download.php?branch=stable.
+
+
+After that, copy the ISO file to your Ventoy Drive.
+
+Instructions on using Gparted are available [here](https://gparted.org/display-doc.php?name=help-manual).
+All the steps using will be covered at: https://clonezilla.org//clonezilla-live-doc.php
```

Comment:
Perhaps place these under the Gparted section? Also what is clonezilla supposed to do here?